### DISCUSSION TITLE
HERO5 BLACK - MOTION CAPTURE CAPABILITIES

### DISCUSSION THREAD

**Discussion by DJCBravo** [gopro/labs#546] [Feature Requests]
If the Hero5 Session has Motion Capture capabilities, it be so great if it's big brother could have the same ability!

Please make https://[gopro.github.io/labs/control/motion/](https://gopro.github.io/labs/control/motion/) possible for Hero5 Black. Please!

**Comment by dnewman-gpsw** [comment]
It will not happen.

**Reply by DJCBravo** [reply]
why?

**Reply by dnewman-gpsw** [reply]
No developers and QA for a platform that old. Team is small, generally only develop Labs for current cameras.

**Reply by DJCBravo** [reply]
So you're sayin there's a chance ; ) 👍
